scripts/waifulib/xcompile.py

```diff
@@ -45,6 +45,49 @@
 
 PSP_ENVVARS = ['PSPDEV', 'PSPSDK', 'PSPTOOLCHAIN']
 
+class iOS:
+	ctx = None
+	simulator = None
+	sdkpath = None
+	target = None
+	
+	def __init__(self, ctx, simulator):
+		self.ctx = ctx
+
+		if simulator == 'true': sdk = "iphonesimulator"
+		else: sdk = "iphoneos"
+
+		path = subprocess.run(["xcrun", "--show-sdk-path", "--sdk", sdk], stdout=subprocess.PIPE)
+
+		self.sdkpath = path.stdout.decode('utf-8')
+		self.sdkpath = str.rstrip(self.sdkpath)
+
+		self.target = '--target=aarch64-apple-ios'
+		if simulator == 'true': self.target += '-simulator'
+	
+	def cflags(self, cxx = False):
+	
+		cflags = []
+	
+		cflags += [ '-isysroot' + self.sdkpath, self.target, '-mios-version-min=12.0', '-Wno-shorten-64-to-32', '-fpic' ]
```

Comment:
Please avoid adding any kind of `-W` flag here.

Same with `-fpic`, it must be enabled already through `enforce_pic` waf tool.